docs/developer-docs/jr-dev-rules.md

```diff
@@ -33,7 +33,7 @@ The following rules apply to all Junior Developers:
 - May not make a new claim while they have any unaddressed tickets
 - Will prioritize resolving open tickets over development of a new set. For additional information on ticket handling, see [Ticket Handling](#ticket-handling), below.
 - Not eligible to participate in collaborations
-- Not eligible to [revise](/guidelines/content/achievement-set-revisions) sets other than ones they've solely authored
+- May only [revise](/guidelines/content/achievement-set-revisions) sets where the Junior Develoepr is the sole author of the existing set.
```

Comment:
```suggestion
- May only [revise](/guidelines/content/achievement-set-revisions) sets where the Junior Developer is the sole author of the existing set.
```

Reply:
Was already fixing it before I saw this comment :D